CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Linkage Checker Enforcer Rule Change Log
+All notable changes to this project will be documented in this file.
+
+## 1.0.1
+* The enforcer rule now works with artifacts having `${os.detect.classifier}` in their dependencies.
```

Comment:
Can you elaborate on this a little, perhaps link to the definition of this property in the Maven docs?

Reply:
Added explanation with a link to os-maven-plugin that defines the variable.